### PR TITLE
Dev redis microsoft.cache 2020 06 01

### PR DESCRIPTION
**Pull request by ghost**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by sspeaks610** [comment]
Private Repo PR Review

https://github.com/Azure/azure-rest-api-specs-pr/pull/1894

**Review by TimLovellSmith — APPROVED** [review]

**Comment by markcowl** [comment]
@sspeaks610 Please read through the PR template in the comments section above and check-off the necessary checks.  We cannot review the PR until this is complete

**Comment by sspeaks610** [comment]
@markcowl Apologies! Should be good now. We already got an ARM review on a private PR here 
https://github.com/Azure/azure-rest-api-specs-pr/pull/1894

File: specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json
Comment (markcowl) [inline specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json:442]: Normally this is called ListBySubscription, rather than List

File: specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json
Comment (markcowl) [inline specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json:776]: Is there another way to list firewall rules?  If not, this can just be List

**Review by markcowl — CHANGES_REQUESTED** [review]
A couple of minor operation naming comments, otherwise LGTM

**Comment by markcowl** [comment]
@sspeaks610 A couple of minor comments, please take a look.  Otherwise, LGTM

**Review by markcowl — APPROVED** [review]

**Comment by markcowl** [comment]
@sspeaks610 Waiting for ci, otherwise LGTM

**Pull request merged**
